srcpkgs/xorgxrdp/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'xrdp'
+pkgname=xorgxrdp
+version=0.2.16
+revision=3
+archs="x86_64"
```

Comment:
If this is only because you can't test, please remove archs=, otherwise explain.